### PR TITLE
Update wechaty.md

### PR DESCRIPTION
**Pull request by su-chang**
1. check the pair of `async` and `await`
2. confirm the return type of instance function

### CODE REVIEW

File: root/api/wechaty.md [hunk 1]
```diff
@@ -157,16 +157,16 @@ wechaty.on('message', (message) => {
 ```javascript
 // Friendship Event will emit when got a new friend request, or friendship is confirmed.
 
-bot.on('friendship', (friendship) => {
-  if(friendship.type() === Friendship.Type.Receive){ // 1. receive new friendship request from new contact
+bot.on('friendship', async (friendship) => {
+  if(friendship.type() === bot.Friendship.Type.Receive){ // 1. receive new friendship request from new contact
```
Comment (windmemory) [inline root/api/wechaty.md]: Please add a space before and after the `(` and `)`.

File: root/api/wechaty.md [hunk 2]
```diff
@@ -157,16 +157,16 @@ wechaty.on('message', (message) => {
...
     const contact = friendship.contact()
     let result = await friendship.accept()
       if(result){
```
Comment (windmemory) [inline root/api/wechaty.md]: ditto

File: root/api/wechaty.md [hunk 3]
```diff
@@ -157,16 +157,16 @@ wechaty.on('message', (message) => {
...
         console.log(`Request from ${contact.name()} is accept succesfully!`)
       } else{
```
Comment (windmemory) [inline root/api/wechaty.md]: space after `else`

File: root/api/wechaty.md [hunk 4]
```diff
@@ -157,16 +157,16 @@ wechaty.on('message', (message) => {
...
         console.log(`Request from ${contact.name()} failed to accept!`)
       }
-      } else if (friendship.type() === Friendship.Type.Confirm) { // 2. confirm friendship
+      } else if (friendship.type() === bot.Friendship.Type.Confirm) { // 2. confirm friendship
```
Comment (windmemory) [inline root/api/wechaty.md]: The indentation is weird here.

File: root/api/wechaty.md [hunk 5]
```diff
@@ -158,18 +158,18 @@ wechaty.on('message', (message) => {
 // Friendship Event will emit when got a new friend request, or friendship is confirmed.
 
 bot.on('friendship', async (friendship) => {
-  if(friendship.type() === bot.Friendship.Type.Receive){ // 1. receive new friendship request from new contact
+  if(friendship.type() === bot.Friendship.Type.Receive) { // 1. receive new friendship request from new contact
```
Comment (windmemory) [inline root/api/wechaty.md]: Still need space after the `if`.

File: root/api/wechaty.md [hunk 6]
```diff
@@ -158,18 +158,18 @@ wechaty.on('message', (message) => {
...
-      if(result){
-        console.log(`Request from ${contact.name()} is accept succesfully!`)
-      } else{
-        console.log(`Request from ${contact.name()} failed to accept!`)
-      }
-      } else if (friendship.type() === bot.Friendship.Type.Confirm) { // 2. confirm friendship
-      console.log(`new friendship confirmed with ${contact.name()}`)
-   }
- })
+    if(result) {
```
Comment (windmemory) [inline root/api/wechaty.md]: ditto

**Comment by lijiarui** [comment]
LGTM!

**Pull request merged**